platform/src/main/java/org/stellar/anchor/platform/paymentobserver/StellarPaymentObserver.java

```diff
@@ -124,8 +122,8 @@ public static Builder builder() {
 
   public static class Builder {
     String horizonServer = "https://horizon-testnet.stellar.org";
-    List<String> accounts = new LinkedList<>();
-    List<PaymentListener> observers = new LinkedList<>();
+    Set<String> accounts = new TreeSet<>();
+    Set<PaymentListener> observers = new HashSet<>();
```

Comment:
Can you share the reason to change from `List` to `Set`? 

Set is un-sorted and the order can be indeterministic. If we are to use `Set`, can we use SortedSet?